app/views/customizable/_form.html.erb

```diff
@@ -27,13 +27,19 @@ See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
 
-<%# locals: form %>
+<%# locals: form, all_fields, only_required #%>
 
 <% form.object.custom_field_values.each do |value| %>
   <%= form.fields_for_custom_fields :custom_field_values, value do |value_form| %>
     <% required = value.custom_field.is_required? %>
     <div class="form--field <%= required ? '-required' : '' %>">
-      <%= value_form.custom_field %>
+      <% if all_fields %>
```

Comment:
going for `all_fields || only_required && required || !only_required && !required` since we need to display all fields or only mandatory fields or only optional fields :)